packages/nimble-components/src/table-column/base/tests/table-column.spec.ts

```diff
@@ -91,7 +91,8 @@ describe('TableColumn', () => {
                     super({
                         cellRecordFieldNames: [],
                         cellViewTag: 'div',
-                        groupHeaderViewTag: tableColumnEmptyGroupHeaderViewTag
+                        groupHeaderViewTag: tableColumnEmptyGroupHeaderViewTag,
```

Comment:
delegated events are a new feature of the table-column base class. I'd expect tests to cover this feature on it's own.

Reply:
Because the event delegation involves the cell view and the column, I have to test this at the table level, not in the base column tests. I only came up with a few tests (plus the cell view test I wrote).